frontend/packages/dev-console/src/components/topology/filters/FilterDropdown.scss

```diff
@@ -0,0 +1,3 @@
+.odc-filter-dropdown__select .pf-c-select__toggle-badge {
+  display: none;
+}
```

Comment:
Any discussion with UX as to why they are going against the PF design to not show the badge?
If so, any issue raised against PF to allow us to configure this via props instead.

Reply:
kept the badge on the dropdown as per UX discussion and removed these styles